### PR TITLE
Ordering over Int

### PR DESCRIPTION
**Pull request by LuuBluum**
Defines an ordering over the integers, with the basic expected properties.

### CODE REVIEW

**Review by mortberg — APPROVED** [review]
Looks good to me. Should we merge? @felixwellen

**Comment by felixwellen** [comment]
Yes. It has been around for a while, so we should be ready to revert if it fails.

File: Cubical/Data/Int/Properties.agda [hunk 1]
```diff
@@ -13,12 +13,69 @@ open import Cubical.Relation.Nullary
 open import Cubical.Data.Empty as ⊥
 open import Cubical.Data.Bool
 open import Cubical.Data.Nat
-  hiding   (+-assoc ; +-comm)
-  renaming (_·_ to _·ℕ_; _+_ to _+ℕ_ ; ·-assoc to ·ℕ-assoc ; ·-comm to ·ℕ-comm ; isEven to isEvenℕ ; isOdd to isOddℕ)
+  hiding   (+-assoc ; +-comm ; min ; max ; minComm ; maxComm)
+  renaming (_·_ to _·ℕ_; _+_ to _+ℕ_ ; ·-assoc to ·ℕ-assoc ;
+           ·-comm to ·ℕ-comm ; isEven to isEvenℕ ; isOdd to isOddℕ)
```
Comment (mortberg) [inline Cubical/Data/Int/Properties.agda:18]:
```suggestion
            ·-comm to ·ℕ-comm ; isEven to isEvenℕ ; isOdd to isOddℕ)
```

**Comment by LuuBluum** [comment]
Oh yeah, I totally forgot I even made this PR.

**Pull request merged**